txt2tags.py

```diff
@@ -82,7 +82,7 @@
 my_url = "https://txt2tags.org"
 my_name = "txt2tags"
 my_email = "jendrikseipp@gmail.com"
-__version__ = "3.7"
+__version__ = "3.7.2"
```

Comment:
No need to change the version. I'll do that before the next release.

Reply:
eliminated